docs/clients/promtail/configuration.md

```diff
@@ -979,43 +1017,47 @@ sync_period: "10s"
 
 ## Example Docker Config
 
+It's fairly difficult to tail docker files on a standalone machine because they are in different locations for every OS.  We recommend the [docker logging driver](../../../cmd/docker-driver/README.md) for local docker installs or docker-compose.
+
+If running in a kubernetes environment, you should look at the defined configs which are in [helm](../../../production/helm/promtail/templates/configmap.yaml) and [jsonnet](../../../production/ksonnet/promtail/scrape_config.libsonnet), these leverage the prometheus service discovery libraries (and give promtail it's name) for automatically finding and tailing pods.  The jsonnet config explains with comments what each section is for.
```

Comment:
I think when we have a docs site, these relative links are going to break. Can you make these absolute permalinks to the Helm config and Jsonnet?

Reply:
is that true? the permalink is gonna take you away from the docs site and park you on github? that seems worse?

Reply:
hmm, yeah, I guess we'll see and we can change it later if we need to.